src/views/articles/subcategory/SingleArticle.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+const Article = () => (
+  <div>
+    <h2>An Article in a subcategory</h2>
```

Comment:
Do we need a separate page to display an article in a subcategory, shouldn't this be the same with a single article's page?

Reply:
Hmm ok. I believe the same component can be used for both viewing a single article and an article in a subcategory as well

Reply:
done